app/views/items/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%@items.each do |item| %>
+<%= image_tag(item.image, size: "150x150" ) %><br/>
```

Comment:
Something nitpicky but that might be cool is I think you can assign size as a variable somewhere in Rails so that we can just call size: preview or size: thumbnail. If you don't do it yourself I might try to figure that out but, I remember learning that inline styles if they can be avoided, are where it's at.